src/views/Login.tsx

```diff
@@ -21,7 +22,25 @@ const Login: React.FC = () => {
   const [isLoading, setIsLoading] = useState<boolean>(false);
 
   const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+  const backendUrl = "http://localhost:8080";
+
+  const loginWithGoogle = async () => {
+    try {
+      window.open(`${backendUrl}/users/google`, "_self");
+
+      const urlParams = new URLSearchParams(window.location.search);
+      const token = urlParams.get('token');
 
+      if (token) {
+        const bearerToken = `Bearer ${token}`;
+        localStorage.setItem('AUTH_TOKEN', bearerToken);
+        dispatch(setIsLoggedIn(true));
+      } 
```

Comment:
Remember to redire t back to home depending on user

Reply:
Yes it was done on the backend